### PR TITLE
rhbz1029734 - ignore invalid po header rather than throw exception

### PR DESCRIPTION
**Pull request by huangp**
If you are concern about the line alignment that's because the source code is using tab not space.

### CODE REVIEW

File: src/main/java/org/fedorahosted/tennera/jgettext/HeaderFields.java [hunk 1]
```diff
@@ -82,15 +86,19 @@ public static HeaderFields wrap(Message message) throws ParseException {
...
 	 * <p>Extracts key:value headers into a {@link HeaderFields}.</p>
-	 * 
+	 *
 	 * <p>
 	 *   The expected format is zero or more lines each in the form
 	 *   <code> "[key]:[value]\n"</code>.
 	 * </p>
-	 * 
+     * <p>
+     *   Other format will be ignored. i.e.
```
Comment (seanf) [inline src/main/java/org/fedorahosted/tennera/jgettext/HeaderFields.java]: You could say "ignored with a log warning".  Not essential.

**Comment by seanf** [comment]
@huangp Was the code previously using a mixture of tab and space, or just tab?

File: src/main/java/org/fedorahosted/tennera/jgettext/HeaderFields.java [hunk 2]
```diff
@@ -99,10 +107,14 @@ public static HeaderFields wrap(String msgstr) throws ParseException {
...
 		for (String entry : entries) {
+            if (entry.trim().isEmpty()) {
+                log.warn("Empty header is found. This entry will be ignored.");  
+                continue;
+            }
 			String[] keyval = entry.split("\\:", 2);
 			if (keyval.length != 2) {
-				throw new ParseException("Could not parse header entry: "
-						+ entry, -1);
+                log.warn("Header entry is not key value pair [{}]. It will be ignored.", entry);
```
Comment (seanf) [inline src/main/java/org/fedorahosted/tennera/jgettext/HeaderFields.java]: Perhaps "not key:value pair" for clarity?

**Comment by huangp** [comment]
previously use tab. my new code is space. I can switch to use tab but more inclined to reformat the whole project afterwards.

**Comment by seanf** [comment]
Well, it is always better to make changes consistent with the existing file contents.  But rather than make the change again, how about we keep this as is, and then have another PR to reformat the whole lot to be consistent with Zanata's conventions?

**Comment by huangp** [comment]
Ok. Amended my commit

**Comment by seanf** [comment]
:+1: apart from the reformatting.

**Pull request merged**